shared/remediations/bash/kernel_module_bluetooth_disabled.sh

```diff
@@ -1,2 +1,3 @@
-# platform = multi_platform_rhel
+# platform = multi_platform_rhel, multi_platform_fedora
+# THIS FILE IS GENERATED. DO NOT EDIT!
 echo "install bluetooth /bin/true" > /etc/modprobe.d/bluetooth.conf
```

Comment:
This should be updated to be idempotent. Should probably make sure that the template makes it idempotent as well.